sdk/bare/common/sys/Logging-Binary-Format.md

```diff
@@ -0,0 +1,40 @@
+# Logging Binary Format
+
+## Overview
+
+The AMDC can dump the logged variables to the UART serial console using either text-based formatting or the binary
+format. This document describes the binary formatting which appears on the UART interface.
+
+## Dumping Variables
+
+Each logged variable must be dumped one at a time (i.e. if the user has 10 slots, each must be dumped one by one).
+
+To start a dump of the Nth variable slot: `log dump bin N`
+
+## Binary Format
+
+All data is little-endian.
+
+### Magic Header (16 bytes)
+
+0x12345678 (repeated four times)
+
+### Number of Samples (4 bytes, integer)
+
+0x00000010
+
+### Data Type (4 bytes, integer)
```

Comment:
Seems like we are wasting 3 bytes here. But that's probably okay since it adds only minimal overhead and does allow future growth for some kind of potential weird homebrew data types (or data compression)

Reply:
Yeah, this could easily be a 1 byte field... But its only 3 bytes of overhead. I'm opting to not change this now.